sdk/errors.go

```diff
@@ -21,6 +22,15 @@ func APIError(err error) *mongodbatlasv2.Error {
 	return &transformedError
 }
 
+// APIErrorToErr returns APIError as instance of Go error or nil if passed error is not an API error.
+func APIErrorToErr(err error) error {
+	apiError := APIError(err)
+	if apiError != nil {
+		return fmt.Errorf("%v", apiError)
```

Comment:
we need this for atlas cli that returns generic errors.
Alternatively we can use Wrap/Unwrap to carry original error but it will also drive a lot of boilerplate.

ApiError can be also stringified to JSON string